internal/controller/runtime/runtime.go

```diff
@@ -0,0 +1,99 @@
+package runtime
+
+/*
+Copyright 2020 Crunchy Data
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	"os"
+	"time"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	"github.com/crunchydata/postgres-operator/internal/controller/postgrescluster"
+	"github.com/crunchydata/postgres-operator/pkg/apis/postgres-operator.crunchydata.com/v1alpha1"
+)
+
+const (
+	// the name of the event recorder created for the postgrescluster controller
+	recorderName = "postgrescluster-controller"
+
+	// default refresh interval in minutes
+	refreshInterval = 60
```

Comment:
If we shouldn't set it smaller than one minute, we can say so in the comment.

```suggestion
	// default refresh interval
	refreshInterval = 60 * time.Minute
```

Reply:
This was just a starting point, so I don't have any specific constraints around this interval at the moment (maybe we make it configurable, as similar settings currently are?).

Reply:
+1 on configurable.

Reply:
Similar to my comment for `workerCount`, we have not yet established our configuration approach/solution for `postgrescluster`, so I was not considering that in-scope for this initial controller implementation.

Reply:
My gripe is with the lack of type. A constant amount of type should be a `time.Duration`.

Reply:
I'm cool with that.  Being that similar configurable settings for `pgcluster` are defined as `int's` within various operator configuration structs (e.g. for `pgo.yaml`), I essentially just had the same mindset here (considering this will likely eventually be configurable).  However, I'm OK with changing this to `time.Duration` until we figure out our configuration approach, and then adjust these settings accordingly.